warehouse/integrations/deltalake/deltalake_test.go

```diff
@@ -1319,6 +1319,75 @@ func TestIntegration(t *testing.T) {
 			}
 		})
 	})
+
+	t.Run("Add columns to table", func(t *testing.T) {
+		tableName := "test_add_existing_columns"
+		namespace := whth.RandSchema(destType)
+
+		loadFiles := []whutils.LoadFile{{Location: "dummy_location"}}
+		mockUploader := newMockUploader(t, loadFiles, tableName, nil, nil, whutils.LoadFileTypeCsv, false, false)
+		ctx := context.Background()
+
+		d := deltalake.New(config.New(), logger.NOP, stats.NOP)
+		warehouse := model.Warehouse{
+			Destination: backendconfig.DestinationT{
+				ID: "test_destination_id",
+				DestinationDefinition: backendconfig.DestinationDefinitionT{
+					Name: destType,
+				},
+				Config: map[string]any{
+					"host":           credentials.Host,
+					"port":           credentials.Port,
+					"path":           credentials.Path,
+					"token":          credentials.Token,
+					"namespace":      namespace,
+					"bucketProvider": whutils.AzureBlob,
+					"containerName":  credentials.ContainerName,
+					"accountName":    credentials.AccountName,
+					"accountKey":     credentials.AccountKey,
+				},
+			},
+			Namespace: namespace,
+		}
+		err := d.Setup(ctx, warehouse, mockUploader)
+		require.NoError(t, err)
+
+		err = d.CreateSchema(ctx)
+		require.NoError(t, err)
+		t.Cleanup(func() {
+			dropSchema(t, d.DB.DB, d.Namespace)
+		})
+		columns := map[string]string{
+			"col1": "int",
+			"col2": "string",
+		}
+
+		err = d.CreateTable(ctx, tableName, columns)
+		require.NoError(t, err)
+
+		columnsToAdd := []whutils.ColumnInfo{
+			{
+				Name: "col1",
+				Type: "int",
+			},
+			{
+				Name: "col3",
+				Type: "int",
+			},
+		}
+
+		err = d.AddColumns(ctx, tableName, columnsToAdd)
+		require.NoError(t, err)
+
+		schema, err := d.FetchSchema(ctx)
+		require.NoError(t, err)
+		require.Contains(t, schema, tableName)
+		require.Equal(t, model.TableSchema(map[string]string{
+			"col1": "int",
+			"col2": "string",
+			"col3": "int",
+		}), schema[tableName])
```

Comment:
Redundant type conversion 
```suggestion
		require.Equal(t, model.TableSchema{
			"col1": "int",
			"col2": "string",
			"col3": "int",
		}, schema[tableName])
```

Reply:
removed